iree/compiler/Conversion/LinalgToSPIRV/LinalgTileAndFusePass.cpp

```diff
@@ -547,12 +624,29 @@ void LinalgTileAndFusePass::runOnOperation() {
         OwningRewritePatternList vectorUnrollPatterns;
         populateVectorUnrollPatterns(context, vectorUnrollPatterns);
         applyPatternsAndFoldGreedily(funcOp, std::move(vectorUnrollPatterns));
+
+        OwningRewritePatternList canonicalizationPatterns;
```

Comment:
Maybe we should move all the vectorization related patterns and application to its own function to make the main function a bit more readable?